tests/server/test_agents_api.py

```diff
@@ -123,7 +123,7 @@ def agent_names(agents: list[dict[str, str]]) -> list[str]:
 async def test_agents_paging(server, client, env_with_agents: None, environment: str, order_by_column: str, order: str) -> None:
     result = await client.get_agents(
         environment,
-        filter={"status": ["paused", "up"]},
```

Comment:
why change this ?

Reply:
I refactored a bit to make it obvious in the `env_with_agents` fixture why everything (except paused agents) is down